blocks/form/__custom/form__custom.css

```diff
@@ -0,0 +1,3 @@
+.form__custom {
+  width: 392px;
```

Comment:
Fixed width is not the best solution. It works terribly when you want to make your layout responsive

Reply:
Correct this shouldn't be fixed. I updated it to be max width and will be changing it to be responsive based on the different screen sizes the client wants.